Doc/whatsnew/3.11.rst

```diff
@@ -493,6 +495,222 @@ Optimizations
   (Contributed by Inada Naoki in :issue:`46845`.)
 
 
+Faster CPython
+==============
+
+CPython 3.11 is on average `1.22x faster <https://github.com/faster-cpython/ideas/blob/main/main-vs-310.rst>`_
+than CPython 3.10 when measured with the
+`pyperformance <https://github.com/python/pyperformance>`_ benchmark suite, on
+Ubuntu Linux. Depending on your workload, the speedup could be 10-60% faster.
```

Comment:
Should we mention compiler too? https://github.com/faster-cpython/ideas/discussions/331